projects/canopy/src/lib/carousel/carousel.component.scss

```diff
@@ -1,5 +1,61 @@
+@import '../../styles/mixins.scss';
+
 .lg-carousel {
   overflow: hidden;
+  position: relative;
+
+  &__autoplay {
+    position: absolute;
+    right: var(--space-sm);
+    z-index: 1;
+    top: var(--space-sm);
+
+    &__button {
+      background-color: var(--color-white);
+      border-radius: 50%;
+      border: none;
+      width: var(--space-xxl);
+      height: var(--space-xxl);
+      font-size: var(--text-fs-3-size);
+      cursor: pointer;
+
+      &[keyboard-focus]:focus {
+        @include lg-outer-focus-outline();
+      }
+    }
+  }
+
+  &__button-timer {
```

Comment:
Looks like the `play` button is not fully centred:
![Screenshot 2021-06-22 at 15 44 33](https://user-images.githubusercontent.com/8397116/122946710-7f926d80-d371-11eb-94ff-2c26b5e09744.png)

Reply:
Same for the pause icon too :)